src/components/Terminal.jsx

```diff
@@ -55,8 +55,6 @@ export default class Terminal extends React.Component {
     if (dangerMode) messageElement.innerHTML = message
     else messageElement.innerText = message
 
-    messageElement.style = 'margin: 0px; line-height: 21px;'
```

Comment:
This change is actually invalid since it compromises current functionality completely, but I was going to completely remove this functionality soon anyhow so we'll let it slide.

Reply:
One principle of this commit is use className and it's own css to style it instead of inline styles.